src/test/java/com/rackspace/ceres/app/services/MetricDeletionServiceTest.java

```diff
@@ -122,11 +122,59 @@ public void testDeleteMetricsByTenantId() {
     metricDeletionService.deleteMetrics(tenantId, "", null,
         Instant.now().minusSeconds(60), Instant.now()).then().block();
 
-    assertViaQuery(tenantId, timeSlotPartitioner.rawTimeSlot(currentTime), seriesSetHash,
-        metricName);
+    //validate data raw
+    assertQueryRawViaQuery(tenantId, timeSlotPartitioner.rawTimeSlot(currentTime), 0);
```

Comment:
Are you asserting boundaries are deleted? E.g. start, end. Also in the middle.

Reply:
I am querying data from 60 seconds earlier to current time and onto them I am asserting data metrics data is deleted or not, keeping its metadata in its place.

Reply:
I mean if you delete "from - to" shouldn't you check that from time stamp has no data anymore as well as to timestamp and also one sample in the middle? This is what I mean by boundary checking.